test/e2e/data/infrastructure-aws/withoutclusterclass/kustomize_sources/addons/csi/data/aws-ebs-csi-external.yaml

```diff
@@ -4,8 +4,8 @@ metadata:
   name: aws-secret
   namespace: kube-system
 stringData:
-  key_id: ""
```

Comment:
Use explict key and access key in csi yaml should not be the best practice.
https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/4247 This support ideally will allow we zero the aws credentials in csi yaml.

Reply:
you could make a change in a separate PR